custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -53,10 +56,19 @@ def mlpiper_configure(self, params):
 
     def _init_mlops(self, params):
 
+        if not self._params["monitor_settings"]:
+            self._mlops_spool_dir = tempfile.mkdtemp()
+            monitor_settings = "spooler_type=FILESYSTEM;directory={};max_files=5;file_max_size=10485760".format(
+                self._mlops_spool_dir
+            )
+        else:
+            monitor_settings = self._params["monitor_settings"]
```

Comment:
I think a safer and cleaner way is:

```suggestion
        monitor_settings = self._params.get("monitor_settings")
        if not monitor_settings:
            self._mlops_spool_dir = tempfile.mkdtemp()
            monitor_settings = "spooler_type=FILESYSTEM;directory={};max_files=5;file_max_size=10485760".format(
                self._mlops_spool_dir
            )
```
(But, your call)